README.md

```diff
@@ -0,0 +1,28 @@
+# Poo Farmer
+
+## Story
+
+It is the **Time Now**, a non-negative, non-zero year about 63 years after the collapse of the global industrial economy, the **Time Before**. Nation states have dissolved into smaller, more local and feudal counties. Although remnants of the technological advancements from the **Time Before** are still present, and in some cases still part of everyday life, most are left to ruin due to the inability to organize broader community support their maintenance at scale.
```

Comment:
Is this supposed to be `feudal countries`? Also, the end of this paragraph is a little confusing

Reply:
No, I am meaning counties. Like smaller sections of land rules by count and countess type people.

Does that still fit?

Reply:
I see yeah. Maybe the ending should be `most are left to ruin due to the inability to organize the community to support their maintenance.`

Reply:
Okay updated this.